configs/_base_/datasets/wider_face.py

```diff
@@ -13,51 +21,61 @@
+        mean=[123.675, 116.28, 103.53],
+        to_rgb=True,
         ratio_range=(1, 4)),
     dict(
         type='MinIoURandomCrop',
         min_ious=(0.1, 0.3, 0.5, 0.7, 0.9),
         min_crop_size=0.3),
-    dict(type='Resize', img_scale=(300, 300), keep_ratio=False),
-    dict(type='Normalize', **img_norm_cfg),
-    dict(type='RandomFlip', flip_ratio=0.5),
-    dict(type='DefaultFormatBundle'),
-    dict(type='Collect', keys=['img', 'gt_bboxes', 'gt_labels']),
+    dict(type='Resize', scale=(300, 300), keep_ratio=False),
+    dict(type='RandomFlip', prob=0.5),
+    dict(type='PackDetInputs')
 ]
 test_pipeline = [
-    dict(type='LoadImageFromFile'),
+    dict(type='LoadImageFromFile', file_client_args=file_client_args),
+    dict(type='Resize', scale=(300, 300), keep_ratio=False),
+    # avoid bboxes being resized
+    dict(type='LoadAnnotations', with_bbox=True),
     dict(
-        type='MultiScaleFlipAug',
-        img_scale=(300, 300),
-        flip=False,
-        transforms=[
-            dict(type='Resize', keep_ratio=False),
-            dict(type='Normalize', **img_norm_cfg),
-            dict(type='ImageToTensor', keys=['img']),
-            dict(type='Collect', keys=['img']),
-        ])
+        type='PackDetInputs',
+        meta_keys=('img_id', 'img_path', 'ori_shape', 'img_shape',
+                   'scale_factor'))
 ]
-data = dict(
-    samples_per_gpu=60,
-    workers_per_gpu=2,
-    train=dict(
+
+train_dataloader = dict(
+    batch_size=32,
+    num_workers=8,
+    persistent_workers=True,
+    drop_last=False,
+    sampler=dict(type='DefaultSampler', shuffle=True),
+    batch_sampler=dict(type='AspectRatioBatchSampler'),
+    dataset=dict(
         type='RepeatDataset',
         times=2,
         dataset=dict(
             type=dataset_type,
-            ann_file=data_root + 'train.txt',
-            img_prefix=data_root + 'WIDER_train/',
-            min_size=17,
-            pipeline=train_pipeline)),
-    val=dict(
-        type=dataset_type,
-        ann_file=data_root + 'val.txt',
-        img_prefix=data_root + 'WIDER_val/',
-        pipeline=test_pipeline),
-    test=dict(
+            data_root=data_root,
+            ann_file='train.txt',
+            data_prefix=dict(img='WIDER_train'),
+            filter_cfg=dict(
+                filter_empty_gt=True, bbox_min_size=17, min_size=32),
+            pipeline=train_pipeline)))
+
+val_dataloader = dict(
+    batch_size=1,
+    num_workers=2,
+    persistent_workers=True,
+    drop_last=False,
+    sampler=dict(type='DefaultSampler', shuffle=False),
+    dataset=dict(
         type=dataset_type,
-        ann_file=data_root + 'val.txt',
-        img_prefix=data_root + 'WIDER_val/',
+        data_root=data_root,
+        ann_file='val.txt',
+        data_prefix=dict(img='WIDER_val'),
+        test_mode=True,
         pipeline=test_pipeline))
+test_dataloader = val_dataloader
+
+val_evaluator = dict(type='VOCMetric', metric='mAP', eval_mode='11points')
```

Comment:
in VOCMetric `use_legacy_coordinate` is a hardcode that defaults to True, which may get some error in Wider_face. Seems it is necessary to check if voc dataset, `use_legacy_coordinate=True`.
Also, I'm not sure use `VOCMetric` here is reasonable. Maybe we can set a `XMLMetrc` as a baseMetric, and set `VOC` and `WiderFace` to inherit it? Please @RangiLyu have a look